Source/Core/Core/FifoPlayer/FifoRecorder.cpp

```diff
@@ -16,18 +15,10 @@
 #include "Core/HW/Memmap.h"
 
 static FifoRecorder instance;
-static std::recursive_mutex sMutex;
-
-FifoRecorder::FifoRecorder() = default;
```

Comment:
Constructors and destructors are typically left defaulted to prevent inlining of construction and teardown code (which is usually not desirable for classes without a guarantee of always being lightweight). It also prevents the need for recompilation of any source files that include it, should any addition to the constructor be necessary, which is why I left is as a defaulted definition instead of removing it outright.

Reply:
I'm confused as to why you're bringing up construction/destruction inlining, since this is only initialized as a global static instance, but sure I'll leave the constructor declaration in to prevent the need for recompiling any other files if we ever change it.

Reply:
> I'm confused as to why you're bringing up construction/destruction inlining, since this is only initialized as a global static instance.

Just because it's only currently used as a global doesn't mean that it won't potentially change in the future.